app/app.js

```diff
@@ -683,6 +687,17 @@ app.post('/snap', [
           });
         }
 
+        // URL timed out.
+        if (err.message.indexOf('ERR_TIMED_OUT') !== -1) {
```

Comment:
```suggestion
        if ((err.message.indexOf('ERR_TIMED_OUT') !== -1) || (err.name ==  'TimeoutError')) {
```

This makes it work on Chromium, too.